tests/apps/courses/test_models_course.py

```diff
@@ -341,11 +336,14 @@ def test_models_course_get_organizations(self):
         # course in the following, the third category will not be linked so we can check that
         # only the organizations linked to the course are retrieved (its name starts with `_`
         # because it is not used and only here for unpacking purposes)
-        *draft_organizations, _other_draft = OrganizationFactory.create_batch(3)
-        *published_organizations, _other_public = OrganizationFactory.create_batch(
-            3, should_publish=True
-        )
-        course = CourseFactory(
+        *draft_organizations, _other_draft = factories.OrganizationFactory.create_batch(
```

Comment:
Sometimes lint rules don't do what they are supposed to do (in this case improve readability).

Reply:
:man_shrugging: